recipes/librdkafka/all/patches/0002-Change-library-targets-and-result-variables-1-5-0.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 3c49a902..1df36a9a 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -209,9 +209,8 @@ if(WITH_ZLIB)
+ endif()
+ 
+ if(WITH_ZSTD)
+-  target_link_libraries(rdkafka PUBLIC ${ZSTD_LIBRARY})
+-  target_include_directories(rdkafka PUBLIC ${ZSTD_INCLUDE_DIR})
+-  message(STATUS "Found ZSTD: ${ZSTD_LIBRARY}")
++  target_link_libraries(rdkafka PUBLIC ${zstd::zstd})
++  message(STATUS "Found ZSTD: ${zstd_LIBRARY}")
+ endif()
+ 
+ if(WITH_SSL)
+@@ -224,7 +223,6 @@ if(WITH_SSL)
+     add_dependencies(rdkafka bundled-ssl)
+   else()
+     find_package(OpenSSL REQUIRED)
+-    target_include_directories(rdkafka PUBLIC ${OPENSSL_INCLUDE_DIR})
+     target_link_libraries(rdkafka PUBLIC OpenSSL::SSL OpenSSL::Crypto)
+   endif()
+ endif()
+@@ -237,8 +235,7 @@ find_package(Threads REQUIRED)
+ target_link_libraries(rdkafka PUBLIC Threads::Threads)
+ 
+ if(WITH_SASL_CYRUS)
+-  target_include_directories(rdkafka PUBLIC ${SASL_INCLUDE_DIRS})
+-  target_link_libraries(rdkafka PUBLIC ${SASL_LIBRARIES})
++  target_link_libraries(rdkafka PUBLIC cyrus-sasl::cyrus-sasl)
```

Comment:
```suggestion
+  target_link_libraries(rdkafka PUBLIC CONAN_PKG::cyrus-sasl)
```

`cyrus-sasl` doesn't have official CMake module or config file, more robust to use `cyrus-sasl::cyrus-sasl`, because you can't predict that upstream will not create official config file with different CMake names, leading to change in upstream recipe and breaking this one.